lib/helper.js

```diff
@@ -71,5 +97,51 @@ module.exports = {
       }
     }
     return null;
+  },
+  /**
+   * this function builds a `form-data` object for each file parameter
+   * @param {Object} fileParams an object with fields data and contentType
```

Comment:
jsdoc can be added for each field in `fileParams`. See PersonalityInsightsV3

Reply:
Will do.